.travis.yml

```diff
@@ -60,21 +54,21 @@ sudo: required
 
 env:
   global:
-  - DISPLAY=:99.0
-  - GECKODRIVER_VERSION=0.21.0
-  - PYTEST_ADDOPTS="-m 'not (chrome or edge or safari or phantomjs)'"
+  - PYTEST_ADDOPTS="-m 'not (edge or safari or phantomjs)'"
+  - GECKO_FALLBACK_VERSION="v0.24.0"
```

Comment:
Let's call this `GECKODRIVER_FALLBACK_VERSION` it's more verbose, but Gecko is the engine and has it's own version.